pkg/install/stack/usercluster-mla/stack.go

```diff
@@ -91,6 +91,8 @@ func (*UserClusterMLA) Name() string {
 }
 
 func (s *UserClusterMLA) Deploy(ctx context.Context, opt stack.DeployOptions) error {
+	opt.Logger.Info("ℹ️ For usercluster-mla deployment, it is recommended to extend helm timeout by using flag --helm-timeout=15m0s...")
```

Comment:
"by using _the_ `--helm-helmout` flag"

Reply:
And if this is so strongly recommended, shouldn't we use a different default timeout for this stack and only issue this warning if the user gave a lower timeout? Currently this reads like "we know this is slow and our timeout is too small, but we're not setting a sensible default either, so please fix our stuf!" ^^

Reply:
>shouldn't we use a different default timeout for this stack
fair point...

Reply:
let's update the default timeout if possible, I agree with @xrstf it would make more sense.

Reply:
I will jump right into it as soon as I'm finished with other incoming PR :)